sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -21,22 +21,28 @@
 import logging
 import math
 import os
-import sys
 import tempfile
 import unittest
 from builtins import range
-
-import avro.datafile
-from avro.datafile import DataFileWriter
-from avro.io import DatumWriter
+import sys
 import hamcrest as hc
+from fastavro.schema import parse_schema
+from fastavro import writer
 # pylint: disable=wrong-import-order, wrong-import-position, ungrouped-imports
 try:
-  from avro.schema import Parse # avro-python3 library for python3
+  from avro.schema import parse # avro library for python2
```

Comment:
Per high-level comment, let's keep Py2 and Py3 Avro codepaths for now.  Previous version on the import may better alight with this requirement.